addons/mail/models/mail_render_mixin.py

```diff
@@ -268,7 +267,7 @@ def _render_eval_context(self):
             'format_amount': lambda amount, currency, lang_code=False: tools.format_amount(self.env, amount, currency, lang_code),
             'format_duration': lambda value: tools.format_duration(value),
             'is_html_empty': is_html_empty,
-            'slug': slug,
+            'slug': self.env['ir.http']._slug,
```

Comment:
This is going to break when installing mail only.

Reply:
Isn't it only `_slugify` that is problematic and not `_slug` (defined in `base`)?

Reply:
At least in whatsapp, changing slugify to slug led to some tests being broken because of implementation stuff. So I bet both are somehow broken. Don't change something that works should be something to keep in mind when the refactoring is both not smaller in LOC nor easier to use.